libs/openblas/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=OpenBLAS
+PKG_SOURCE_VERSION:=e55ec82bb92338d09ecd77357da3fcdfac0a7902
+PKG_VERSION:=0.3.9-$(PKG_SOURCE_VERSION)
+PKG_RELEASE:=1
+#PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+#PKG_SOURCE_URL:=https://github.com/xianyi/OpenBLAS/archive/
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2020-04-19
+PKG_SOURCE_URL:=https://github.com/martin-frbg/OpenBLAS.git
+#PKG_HASH:=28cc19a6acbf636f5aab5f10b9a0dfe1
+PKG_BUILD_DIR:=$(BUILD_DIR)/OpenBLAS-$(PKG_VERSION)
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openblas
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=An optimized BLAS library
+  URL:=https://www.openblas.net/
+endef
+
+define Package/bridge/description
+  OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version
+endef
+# Package build instructions; invoke the GNU make tool to build our package
+define Build/Compile
+        $(MAKE) -C $(PKG_BUILD_DIR) \
+	   TARGET='MIPS24K' \
```

Comment:
do you intend to support openblas only for MIPS24K?
if so, maybe you should limit the depends a bit;
example how to do it per-arch: `DEPENDS:=@(i386||x86_64||arm||armeb||aarch64||powerpc||mips||mipsel)`
you can probably just put `mips`

though it may make sense to try to target all archs if possible [somehow]
otherwise, a package [like this] may not make sense for just 1 arch

Reply:
I want to build for all the archs.
24K Is the one that I am interesting to compile on with `docker run --rm -it openwrtorg/sdk:ramips-mt76x8-19.07.2`.
The issue Is that Openblas for cross-compiling requires `TARGET` as you can see from https://github.com/xianyi/OpenBLAS/blob/develop/README.md#cross-compile.
And for `TARGET` we need to use a value [from this list](https://github.com/xianyi/OpenBLAS/blob/develop/TargetList.txt)

Reply:
See also https://github.com/xianyi/OpenBLAS/issues/2570